c7n/resources/vpc.py

```diff
@@ -44,6 +44,39 @@ class resource_type(query.TypeInfo):
         id_prefix = "vpc-"
 
 
+@Vpc.action_registry.register('modify')
+class ModifyVpc(BaseAction):
+    """Modify vpc settings
+    """
+
+    schema = type_schema(
+        'modify',
+        **{'dns-hostnames': {'type': 'boolean'},
```

Comment:
its unfortunate, but for consistency (at least pair wise between matching filter/action) we should probably match the filter on all of these attributes, and omit the hyphens.